CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ set(CLIENT_VERSION_MINOR 99)
 set(CLIENT_VERSION_BUILD 0)
 set(CLIENT_VERSION_RC 0)
 set(CLIENT_VERSION_IS_RELEASE "false")
-set(COPYRIGHT_YEAR "2025")
+string(TIMESTAMP COPYRIGHT_YEAR "%Y" UTC)
```

Comment:
In commit "build: let CMake determine the year" (dfeac76b951fc2d28e0832b176f3c8ad595697f1)

Doesn't this make the build nondeterministic?

Reply:
No; SOURCE_DATE_EPOCH, which is set by the guix build, is respected: https://github.com/bitcoin/bitcoin/pull/32445#pullrequestreview-2824986604